Sources/HttpResponse.swift

```diff
@@ -22,30 +22,31 @@ public protocol HttpResponseBodyWriter {
 
 public enum HttpResponseBody {
     
-    case json(AnyObject)
+    public enum RawDataType {
+        case any
+        case json
+    }
+
+    static func json<Body: Encodable>(_ body: Body) throws -> HttpResponseBody {
```

Comment:
```suggestion
    static func json(_ body: Encodable) throws -> HttpResponseBody {
```
The generic parameter is not used